### DISCUSSION TITLE
Next Step?

### DISCUSSION THREAD

**Discussion by MauriceB13** [bemasher/rtlamr#172] [Q&A]
I am in the need of a little guidance here. I have RTL-SDR working. However, I can't seem to figure out how to "tune in" to my meter and then view data from it? How do I turn those squelches into data?

Thanks,
Ray

**Comment by bemasher** [comment]
What have you tried so far? Rtlamr is designed so that it doesn't require much tuning, you should be able to run it and receive data by only specifying message type.

**Reply by MauriceB13** [reply]
Hi @bemasher , I have run RTL_TCP. it shows it is tuned to 100000000 Hz and is listening. I then run RTL_SDR using "rtlamr -msgtype=all -format=json -msgtype=idm,scm+". It shows it connected to the RTL_TCP session. See images below.

![rtl_tcp](https://user-images.githubusercontent.com/77756698/107557555-c55f4e00-6ba7-11eb-84e1-44deef4a8d02.png)

![rtlamr](https://user-images.githubusercontent.com/77756698/107557567-ca240200-6ba7-11eb-966e-5a18c1f9557f.png)

It's at this point I'm lost. How do I see devices and data?

Thanks,
Ray

**Reply by bemasher** [reply]
Ray, you should only specify msgtype once. The second use of msgtype is overriding the first use.

If you have supported meters in your area, you will see messages printed to standard out.

**Reply by MauriceB13** [reply]
Ahhhh....OK, I fixed that so only using -msgtype=idm,scm+ now. @bemasher when you say "standard out", you mean in the same DOS window?

**Reply by bemasher** [reply]
Bingo. It'll show in the same window you ran rtlamr in.

**Reply by MauriceB13** [reply]
@bemasher Awesome! So I have it running and nothing is showing up after 1-1/2 hours. Is it possible RTL isn't able to pick up this meter or should I hard code a different frequency? It appears it is looking in the 912,600,155 band.

![rtlamr](https://user-images.githubusercontent.com/77756698/107575033-5e4c9400-6bbd-11eb-80a6-bb05e356ad91.png)

Also, my antenna is in a window in the back of my house. My meter is toward the front of my house. I'd say they are less than 100 feet from each other, not in line of sight. I would think my antenna could pick it up?

This is a picture of my meter:

![20210112_112931](https://user-images.githubusercontent.com/77756698/107576145-bfc13280-6bbe-11eb-9462-86a3b5ae45ef.jpg)

Thanks,
Ray

**Reply by bemasher** [reply]
Unfortunately, your meter is not supported.

**Reply by MauriceB13** [reply]
@bemasher is this because nothing is showing up or have you run into this meter before? Maybe it's on a different frequency?

**Reply by bemasher** [reply]
I have run into this meter before. https://github.com/bemasher/rtlamr/issues?q=focus

**Reply by MauriceB13** [reply]
I believe the operating frequency of my meter is 2.400-2.483 GHz. It may be different than what is mentioned under the issues. I've tried different parameters to set the frequency for rtl_tcp. What is the correct way to set this parameter?

rtl_tcp -centerfreq=2400M

or

rtl_tcp -c=2400M

I can see in SDRSharp that there is something in this range.

**Reply by bemasher** [reply]
Rtl-sdr compatible dongles do not typically function above 1.7Ghz. You would set the frequency in rtlamr with `-centerfreq` if you really wanted to try listening at that frequency. In any case, there are no known Focus-flavored meters that are supported by rtlamr.
